jarviscli/plugins/password_generator.py

```diff
@@ -0,0 +1,25 @@
+from plugin import plugin
+from colorama import Fore
+import requests
+
+@plugin("generate_password")
+
+def generate_password(jarvis, s):
+    length = int(jarvis.input("Desire password length: "))
+    inp = str(jarvis.input("Would you like to exclude numbers? (Y/N) : "))
+    if(inp == 'Y'):
+        exclude_numbers = True
+    else:
+        exclude_numbers = False
+
+    inp = bool(jarvis.input("Would you like to exclude special characters? (Y/N) : "))
```

Comment:
I'm fairly certain this would always evaluate to true as any string is a truthy value.